src/plugins/launch/slurm/launch_slurm.c

```diff
@@ -590,6 +590,53 @@ extern int launch_p_create_job_step(srun_job_t *job, bool use_all_cpus,
 	return SLURM_SUCCESS;
 }
 
+static char *_find_quote_token(char *tmp, char *sep, char **last)
+{
+	char *start;
+	int i, quote_single = 0, quote_double = 0;
+
+	xassert(last);
+	if (*last)
```

Comment:
Stick with the style and always use `{` and `}`

Reply:
this is code copied verbatim from slurm. this patch/commit goes away once they fixed it properly